test_fixtures/strong_mode/web/generic_component.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:over_react/over_react.dart';
+
+@Factory()
+UiFactory<GenericStatelessProps> GenericStateless;
+
+@Props()
+class GenericStatelessProps extends UiProps {}
+
+@Component()
+class GenericStatelessComponent<T extends GenericStatelessProps> extends UiComponent<T> {
```

Comment:
We should add a test fixture for a component with generic state as well, to fully test the generation of `typedStateFactory` in that scenario.